xml/depl_maintenance.xml

```diff
@@ -2180,5 +2180,22 @@ migrate</screen>
       </para>
      </step>
   </procedure>
- </sect1>
+ </section>
+ <xi:include href="operations-maintenance-database_maintenance.xml"/>
+ <section xml:id="sec-depl-maintenance-fernet-tokens">
+  <title>Rotating Fernet Tokens</title>
+  <para>
+     Fernet tokens should be rotated frequently for security purposes.
+     It is recommended to setup this task as a cron job in
+     <literal>/etc/cron.weekly/openstack-keystone-fernet</literal>
+     on the server where the keystone service is running,
+     with the following content:
+  </para>
+  <screen>
+  #!/bin/bash
+  su keystone -s /bin/bash -c "keystone-manage fernet_rotate"
+
+  /usr/bin/keystone-fernet-keys-push.sh 192.168.81.168; /usr/bin/keystone-fernet-keys-push.sh 192.168.81.169;
+  </screen>
```

Comment:
Also add the following ...

"The IP addresses in the above example, i.e. 192.168.81.168 and 192.168.81.169, are the IP addresses of the other two nodes of a three-node cluster. If the other nodes have different IP addresses, you should use them accordingly."